Lecture4/Task4/predefinedExceptions/SelectedVoucherTypeException.java

```diff
@@ -0,0 +1,15 @@
+package predefinedExceptions;
+
+public class SelectedVoucherTypeException extends Throwable {
+
+    private String message;
+
+    public SelectedVoucherTypeException(String message) {
+        this.message = message;
+    }
+
+    @Override
+    public String getMessage() {
+        return message;
```

Comment:
why do you think you need to override default getMessage()?

Reply:
fixed indirectly in Tasks1-4-fixed package